src/components/ListView.js

```diff
@@ -10,10 +10,10 @@ export const ListView = ({ items, containerRef }) => {
     {
       items.map(item => (
         <div class='item' data-selectable data-title={item.title} data-selected-key={item.title} key={item.title}>
-          <div class='info'>
+          <a class='info' href={`/article/en/${item.title}`}>
```

Comment:
👍 I would love to see more opportunities for using html markup over programmatic.

I have tried removing all the relevant event here, it's only working when you `Tab and Enter` or `click' on the desktop browser, doesn't work in KaiOS.

does it work for you?

Reply:
> 👍 I would love to see more opportunities for using html markup over programmatic.
> 
> I have tried removing all the relevant event here, it only works when you `Tab and Enter` or `click' on the desktop browser, doesn't work in KaiOS.
> 
> does it work for you?

I'll get my emulator working on my Linux machine soon to test the changes further..

Reply:
bah, this change was intended for links to work on the Search page.. it's already implemented [here](https://github.com/wikimedia/wikipedia-kaios/blob/f831a31b165b4ba2288d1e11a93adb0b1cea05b0/src/components/Search.js#L32).